patches/log-sse42-status.patch

```diff
@@ -0,0 +1,98 @@
+Reintroduces logging of SSE 4.2 support, but in flag-controlled manner.
+
+From: nobody <nobody@nowhere>
+
+
+---
+ .gitignore          |    2 ++
+ src/Crc32C.cc       |    8 ++++++++
+ src/Crc32C.h        |    6 ++++++
+ src/OptionParser.cc |   12 ++++++++++++
+ 4 files changed, 28 insertions(+)
+
+diff --git a/.gitignore b/.gitignore
+index 4f80bbb8..0f86a172 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -13,6 +13,8 @@
+ 
+ /install/
+ /bindings/java/build/
++/bindings/python/*_pb2.py
++/bindings/python/*_pb2_grpc.py
+ /deps/
```

Comment:
Is this patch carrying along some other changes you have made just for converting RAMCloud to public?  Not that tweaks to .gitignore are a big issue.

Reply:
So for last few months, we've been converting some proto buffs and grpc api's in RAMCloud into compatible py files via a script (primarily to view zookeeper fields in the python standalone tests). These are autogenerated py files from protos. I don't want someone to accidentally commit them. That's why I added them to the ignore list. This part is arguably unrelated to the sse42 issue.